modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/modals/DeleteAPIEndpointModalContent.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayModal from '@clayui/modal';
+import {fetch, openToast} from 'frontend-js-web';
+import React from 'react';
+
+interface DeleteAPIEndpointModal {
```

Comment:
@cgmonte I think you forgot to end the name with "Props".

Reply:
Yes, I did!